rails-commenting.rb

```diff
@@ -6,14 +6,14 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1)Defines the controller which is inherited from ApplicationController.
 class BlogPostsController < ApplicationController
   def index
-    # ---2)
+    # ---2)retrieves all the current resources in the database
     @posts = BlogPost.all
   end
 
-  # ---3)
+  # ---3) Creates a form to enter informsation for a new resource. *this does not disply or show a new resource just creates the form to enter the information*
```

Comment:
The `show` controller method is used to display the details of one specific item! I think you meant to put this answer on line 21 for `new`!